app/controllers/benefits/CompanyBenefitController.scala

```diff
@@ -45,6 +47,8 @@ class CompanyBenefitController @Inject()(
   override implicit val partialRetriever: FormPartialRetriever)
     extends TaiBaseController with JourneyCacheConstants with UpdateOrRemoveCompanyBenefitDecisionConstants {
 
+  private val logger = Logger(this.getClass)
+
```

Comment:
Created a new logger but it's not being used, this needs to be added to conf.xml and logback.xml, try not to add genuine exceptions into here, as this logger is probably not tracked by OS team

Reply:
I mean, it is being used (admittedly only in one case) - Line:142.

But yeah, I can add it to those files.

Reply:
It's getting logged on Line:142 but you'll never be able to see it until its added to the logger configs